.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+*Preferable*
+Gist/Repository with sample code
+
+*Alternatively*
+Steps to reproduce the behavior:
```

Comment:
Bold?

Reply:
Maybe? Depends, I suppose, on whether my comment on the previous item makes sense, or if changes should be made before accepting.

Reply:
Your previous comment makes sense, this is fine as-is.